src/shared_modules/content_manager/tests/unit/executionContext_test.cpp

```diff
@@ -47,17 +47,14 @@ TEST_F(ExecutionContextTest, TestDefaultFolderWhenThereIsNoConfigurationForTheOu
     // Remove the output folder if exists
     removeOutputFolderIfExists(GENERIC_OUTPUT_FOLDER_PATH);
 
-    const auto expectedCapturedOutput = "Created output folder: \"" + GENERIC_OUTPUT_FOLDER_PATH + "\"\n";
-
     m_spUpdaterBaseContext->configData.erase("outputFolder");
 
+    // Start silence stdout
     testing::internal::CaptureStdout();
```

Comment:
I understand that the goal of this is to have a cleaner test execution report. But, when we replace the std::cout with the logging mechanism we have to update the test by removing these lines. With this in mind, I would rather have, temporarily a report with some extra printing, than modify the tests in the future.

Reply:
I will remove the silence from `std::cout`

Reply:
done